src/phoenix/experimental/evals/models/openai.py

```diff
@@ -254,6 +257,7 @@ def verbose_generation_info(self) -> str:
         return f"OpenAI invocation parameters: {self.public_invocation_params}"
 
     async def _async_generate(self, prompt: str, **kwargs: Any) -> str:
+        self.reload_client()
```

Comment:
If it's possible to only re-load the client at the top of generate and classify that might be the least impactful?

Reply:
We'll also need to reload in `run_evals` but it's a little bit more circuitous there since each eval call can be associated with its own underlying model.